jee-commons/src/main/java/biz/paluch/jee/commons/BeanLookup.java

```diff
@@ -136,5 +153,14 @@ public static BeanManager beanManager() {
     public static void reset() {
         lookupStrategy = new BeanManagerLookupStrategy();
     }
+    
+    private static BeanManager beanManagerLookup() {
+        for(String jndiName : BEANMANAGER_JNDI_NAMES) {
+            BeanManager beanManager = NamingLookup.doLookup(jndiName);
```

Comment:
Just picking the first exception sounds a bit random to me. I agree that NamingLookup should throw an exception, what do you think about using a logging API and throw those exceptions as a warning?